pkg/cvo/availableupdates.go

```diff
@@ -160,6 +167,7 @@ func (optr *Operator) setAvailableUpdates(u *availableUpdates) {
 	if u != nil && (optr.availableUpdates == nil ||
 		optr.availableUpdates.Upstream != u.Upstream ||
 		optr.availableUpdates.Channel != u.Channel ||
+		optr.availableUpdates.Architecture != u.Architecture ||
```

Comment:
What would we log?  Something like "bumping LastSyncOrConfigChange because..." and then "upstream changed from A to B", "channel changed from A to B", or "we successfully synchronized available updates"?

Reply:
Never mind. I misunderstood the purpose of this